### PR TITLE
schema-example

### PR DESCRIPTION
**Pull request by kghamilton89**
# 5486

Refactor of CSV schema example page. Removed unclear language, expanded example explanation.

### CODE REVIEW

File: parsers/csv/examples/basic.md [hunk 1]
```diff
@@ -10,32 +10,49 @@ Measurement Time,Sensor Name,Sensor Model,Temperature,Humidity,Pressure
...
+
+`Timestamp Pattern: yyyy-MM-dd HH:mm`
+
+### Schema
 
 ```java
+# select parameters
 select("#row=2-*").select("#col=4-*").
 addSeries().
+
```
Comment (rodionos) [inline parsers/csv/examples/basic.md]: This breaks code, I think. Notice that functions are chained with dot. It's better to move comments out of the code. You could say that the first line selects cells to process, whereas the subsequent lines are use to generate the series command (link to series command in network api).

File: parsers/csv/examples/basic.md [hunk 2]
```diff
@@ -10,32 +10,49 @@ Measurement Time,Sensor Name,Sensor Model,Temperature,Humidity,Pressure
...
-  * Set the entity name to the value of the cell located in the current row, 2nd column (Sensor Name);
-  * Set the tag `model` to the value of cell located in the current row, 3rd column (Sensor Model);
-  * Set the timestamp to the value of the cell located in the current row, 1st column (Measurement Time). The text value is parsed using the **Timestamp Pattern**;
-  * Set the series value to the value of the current cell.
+* `select` each row starting with the second row until the last row `"#row=2-*"`.
+* `select` each column in the second row starting with the forth column until the last column `"#col=4-*"`.
+
+**`series` Parameters**
+
+> `cell` selection is defined by the pattern `(row,col)`
```
Comment (rodionos) [inline parsers/csv/examples/basic.md]: this changes the meaning, not sure what it now means. Perhaps there is a better way to illustrate this, by copying the text into a spreadsheet program that displayed row and column numbers in headers and use arrows to explain the concepts.

File: parsers/csv/examples/basic.md (hunk 1 above)
Comment (kghamilton89) [reply parsers/csv/examples/basic.md]: When using `#` to escape it does break the code, but using `/*` and `*/` to escape still works: https://nur.axibase.com/csv/configs/edit.xhtml?configName=test#testBlock

Comment (kghamilton89) [reply parsers/csv/examples/basic.md]: Measurement Time,Sensor Name,Sensor Model,Temperature,Humidity,Pressure
2015-10-15 00:00,Sensor-0001,PV120000-XG1,       35.5,    40.0,     760
2015-10-15 00:00,Sensor-0020,PV120000-XG1,       20.4,    60.8,     745

File: parsers/csv/examples/basic.md (hunk 2 above)
Comment (kghamilton89) [reply parsers/csv/examples/basic.md]: Removed. Visual explanation added.

**Comment by rodionos** [comment]
To explain selectors, we need to use indexed columns:

image

This way the user can understand how selectors operate and how references to cells make sense.
Maybe we should even add a section on how to develop the schema from a indexed spreadsheet view I pasted (this one is from Excel).

**Comment by kghamilton89** [comment]
@rodionos  Can I add index numbers to the images I've got here? I think they do a good job of showing which cells are selected but I agree that numbers would be even more helpful.

**Comment by axibase-git** [comment]
Yes, adding indexes to screenshots is viable solution.

On Jul 26, 2018, at 1:01 PM, KG Hamilton <notifications@github.com> wrote:

@rodionos <https://github.com/rodionos> Can I add index numbers to the images I've got here? I think they do a good job of showing which cells are selected but I agree that numbers would be even more helpful.

**Comment by kghamilton89** [comment]
Index added to images. I think there is no way to misunderstand them now.

**Comment by rodionos** [comment]
@kghamilton89 - The screenshots show incorrect row indexes. The first row (with header) must start with `1`. Since you'll be redoing them anyways, please adopt our schema colors (see /series/search page).

**Comment by kghamilton89** [comment]
* I don't have Excel on my Linux machine and Excel Online doesn't have the complete range of settings, thus I was unable to change A,B,C,etc columns to 1,2,3,etc. I re-did all screenshots in Libre.
* Colors now conform to standard.

**Pull request merged**